.ldrelease/publish-github-actions-metadata.sh

```diff
@@ -5,11 +5,15 @@ set -ev
 
 # Read from the command line so we can debug this script. Defaults to releaser env variable.
 RELEASE_VERSION=${1:-$LD_RELEASE_VERSION}
+GITHUB_TOKEN=${2:-"${LD_RELEASE_SECRETS_DIR}/github_token"}
 RELEASE_NOTES="$(make echo-release-notes)"
-GITHUB_TOKEN="${LD_RELEASE_SECRETS_DIR}/github_token"
 
-# install gh cli so we can create a release later
-brew install gh
+# install gh cli so we can create a release later https://github.com/cli/cli/blob/trunk/docs/install_linux.md
+curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
```

Comment:
I can't test this locally because this is ubuntu specific so I don't have high confidence in this.